testing/simulated/malicious_consensus_test.go

```diff
@@ -71,9 +75,40 @@ func (s *SimulatedSuite) TestFinalizeBlock_BadBlock_Errors() {
 	)
 	s.Require().NoError(err)
 
+	// Sign a malicious transaction that is expected to fail.
+	maliciousTx, err := gethtypes.SignNewTx(
+		simulated.GetTestKey(s.T()),
+		gethtypes.NewCancunSigner(big.NewInt(int64(s.TestNode.ChainSpec.DepositEth1ChainID()))),
+		&gethtypes.DynamicFeeTx{
+			Nonce:     0,
+			To:        &gethcommon.Address{1},
+			Value:     big.NewInt(100000000000),
+			Gas:       21000,
+			GasTipCap: big.NewInt(10000000),
+			GasFeeCap: big.NewInt(10000000),
+			Data:      []byte{},
+		},
+	)
```

Comment:
@rez may be skill issue but I do not see why this tx is malicious or should fail. Can you clarify (maybe comment in code)

Reply:
wait I guess it is `max fee per gas less than block base fee` (see final assert)